oz/core/actions.py

```diff
@@ -107,7 +119,15 @@ def server():
             signal.signal(signal.SIGTERM, functools.partial(_shutdown_handler, http_srv))
             signal.signal(signal.SIGINT, functools.partial(_shutdown_handler, http_srv))
 
-        tornado.ioloop.IOLoop.instance().start()
+        if server_type == "asyncio":
```

Comment:
Ah, I see what you mean. I'm not sure what the proper fix is here. Maybe we just capture signals if the default ioloop is used?